pytest_fixtures/broker.py

```diff
@@ -108,19 +108,44 @@ def rhel77_contenthost_class(request):
 @pytest.fixture
 def satellite_latest():
     """A fixture that provides a latest Satellite"""
+    version_args = dict(
+        deploy_sat_version=settings.server.version.get('release', ''),
+        deploy_snap_version=settings.server.version.get('snap', ''),
+    )
+
     with VMBroker(
-        host_classes={'host': Satellite}, workflow=settings.server.deploy_workflow
+        host_classes={'host': Satellite}, workflow=settings.server.deploy_workflow, **version_args
     ) as sat:
         yield sat
 
 
 @pytest.fixture
 def capsule_latest():
     """A fixture that provides an unconfigured latest Capsule"""
-    with VMBroker(host_classes={'host': Capsule}, workflow=settings.capsule.deploy_workflow) as cap:
+    version_args = dict(
+        deploy_sat_version=settings.server.version.get('release', ''),
+        deploy_snap_version=settings.server.version.get('snap', ''),
+    )
```

Comment:
this is repeated above, might be best as a constant at the global level of this file

Reply:
ack - I think with #8662 taken into account, this would actually call the get_sat_version helper that takes into account the 'default' satellite instance.